Models/PersonListPart.cs

```diff
@@ -3,8 +3,8 @@
  * by their sex.
  * It's OK to place the part and the part record into the same code file if neither of them are particularly big.
  * Make sure to read the corresponding documentation:
- * http://docs.orchardproject.net/Documentation/Creating-a-module-with-a-simple-text-editor
- * http://docs.orchardproject.net/Documentation/Writing-a-content-part
+ * http://docs.orchardcore.net/projects/O1/en/latest/Documentation/Creating-a-module-with-a-simple-text-editor

```

Comment:
I think it's better now to add these URLs with https instead of http. So, please, replace the http everywhere (not just the http://docs.orchardcore.net ones) to https.

Reply:
In my practice project only 'http://localhost...' works for me, so I left it this way in Controllers\YourFirstOrchardController.cs. Should I change this also?

Reply:
( And owin.org (Services\OwinMiddleware.cs) does not work well with https. )

Reply:
Don't change it "everywhere" just where it makes sense, of course. :). So, yeah, good point, you can keep the http in YourFirstOrchardController.cs and in OwinMiddleware.cs.
But I can see that you changed the URLs in the OrchardHUN.TrainingDemo.csproj too. This means Visual Studio cannot open this project because of the incorrect MSBuild XML namespace.

Reply:
It was an accident, I noticed that and pushed the original csproj immediately. : )

Reply:
We have 2 csproj files: OrchardHUN.TrainingDemo.csproj and OrchardHUN.TrainingDemo.Tests.csproj.
I'm referring to the first one, which still contains https.

Reply:
Oh I see, thank you, and sorry.